alts/shared/utils/asyncssh.py

```diff
@@ -19,34 +25,48 @@ def __init__(
         timeout: Optional[int] = None,
         client_keys_files: Optional[List[str]] = None,
         known_hosts_files: Optional[List[str]] = None,
-        preferred_auth: Optional[Union[str, List[str]]] = None,
+        env_vars: Optional[Dict[str, Any]] = None,
         disable_known_hosts_check: bool = False,
         ignore_encrypted_keys: bool = False,
-        env_vars: Optional[Dict[str, Any]] = None,
+        keepalive_interval: int = 0,
+        keepalive_count_max: int = 3,
```

Comment:
AFAIK this is a default value in `asyncssh`, any particular reason adding it separately? Also, what will happen if the command/script is running for a long period of time, will it stop sending keepalive events?

Reply:
I added it separately in case we would change those values, because by default, `asyncssh` doesn't send `keepalive` messages.
`asyncssh` sent `keepalive` messages the whole connection time, if I'm not wrong